### ISSUE TITLE
Parsing issue with @includedoc and @startuml

### ISSUE THREAD

**Issue by kanje** [doxygen/doxygen#7015]
**Version**: 1.8.15
**Environment**: Linux (Fedora 30)

I am trying to include a PlantUML diagram to a `@page` or `@mainpage` using an `@include{doc}` command. See the [minimal example](https://github.com/doxygen/doxygen/files/3232253/bug-minimal-example.tar.gz) attached. This used to work before, but after upgrading my system I get this output:
```
Example.pu:2: warning: Unexpected new line character
Running PlantUML on generated file /.../html/inline_umlgraph_1.pu
Error line 1 in file: /.../html/inline_umlgraph_1.pu
Some diagram description contains errors
```

`inline_umlgraph_1.pu` is a corrupted version of `Example.pu` passed to `@include{doc}`. Looks like Doxygen throws away the first line.
```
diff Example.pu html/inline_umlgraph_1.pu
2d1
< #Green:Hello;
```

**Comment by albert-github** [comment]
Found a workaround:
At the `@startuml` line add an extra argument (e,g. main) this will be seen as the output file name to be used (so here main.png).
Note that this argument has to be different for each unique `@startuml` otherwise the resulting image are overwritten.


Just a side note: as far as I can see that you did do some command conversions as `\include{doc}` didn't exist before 1.8.15 only `\includedoc` existed, but this is not the cause of the problem.

**Comment by kanje** [comment]
> Found a workaround:
> At the `@startuml` line add an extra argument (e,g. main) this will be seen as the output file name to be used (so here main.png).
> Note that this argument has to be different for each unique `@startuml` otherwise the resulting image are overwritten.

This seems to work. Thanks! It this approach backwards compatible, i.e. work with older Doxygen versions?

> Just a side note: as far as I can see that you did do some command conversions as `\include{doc}` didn't exist before 1.8.15 only `\includedoc` existed, but this is not the cause of the problem.

The original code uses `@includedoc`. I have used `@include{doc}` in the minimal example to see if this resolves the issue, and it remained like this.

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7016

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of a release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.16.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
